simplified-metrics/src/main/java/resources/META-INF/services/org.nypl.simplified.crashlytics.api.CrashlyticsServiceType

```diff
@@ -0,0 +1 @@
+org.nypl.simplified.metrics.api.MetricServiceType
```

Comment:
This file isn't right. The name of the file must be the name of the interface you're passing to `ServiceLoader`, and the line(s) within the file name implementations of that interface. What you're saying here is that you want to call: `ServiceLoader.load(CrashlyticsServiceType.class)` and you want to system to call `new MetricServiceType()` as a result. Probably not what you intended. :smile: 

The file should be called `org.nypl.simplified.metrics.api.MetricServiceType`, and it should contain one like: `org.nypl.simplified.metrics.MetricService`.

There's a further problem here in that the `MetricService` implementation requires an argument to the constructor, and `ServiceLoader` can't handle that. You need to expose a separate factory.

Let me know if you need a hand with this.